shared/src/metabase/shared/models/visualization_settings.cljc

```diff
@@ -420,11 +420,12 @@
   (set/map-invert db->norm-param-ref-keys))
 
 (def ^:private db->norm-table-columns-keys
-  {:name     ::table-column-name
+  {:name      ::table-column-name
    ; for now, do not translate the value of this key (the field vector)
-   :fieldref ::table-column-field-ref
-   :fieldRef ::table-column-field-ref
-   :enabled  ::table-column-enabled})
+   :fieldref  ::table-column-field-ref
+   :field_ref ::table-column-field-ref
+   :fieldRef  ::table-column-field-ref
```

Comment:
A suggestion if you have to do more of this in the future:

It might actually be a little easier just to iterate thru all the keys in the map, normalize them, and then map them to the renaming map. I was going to suggest [CSK](https://github.com/clj-commons/camel-snake-kebab) but I guess that wouldn't work for both `:fieldref` **and** `:field-ref`. 

Something like this might work tho:

```clj
(defn- normalize-key [k]
  (keyword (str/lower-case (str/replace (name k) #"_-" ""))))

(defn normalized-rename-keys [m]
  (let [k->norm {:fieldref ::table-column-field-ref}]
    (into {}
          (map (fn [[k v]]
                 [(get k->norm (normalize-key k) k)
                  v]))
          m)))

(normalized-rename-keys {:field_ref 1, :can_count 2}) ; => {::table-column-field-ref 1, :can_count 2}
```